src/modules/error/index.ts

```diff
@@ -1,8 +1,10 @@
 import { Module, VuexModule, getModule } from 'vuex-module-decorators'
-import store from '@/store'
+import { getVuexStore } from '@/store'
 
 import errorStore from './store'
 
+const store = getVuexStore()
+
 @Module({ dynamic: true, namespaced: false, store, name: 'errorModule' })
```

Comment:
Yeah this is what I tried initially. Something was getting corrupt but I had to test all the way through to the payment summary to see the error. YMMV.